schema.ts

```diff
@@ -854,6 +854,8 @@ export interface Identity extends IdentityDefinition {
 export interface PropertyIdentifierDetails {
   /**
    * An optional label for the property. This will be used in locations where the label appears with the property.
+   *
+   * If set to '', the label will be omitted.
```

Comment:
Or not present, right?

Reply:
for now the default is that labels always show though we have ongoing discussions on whether that should be true

Reply:
will update this if we actively change that decision